app/admin/prizes.rb

```diff
@@ -5,30 +5,28 @@
 
   controller do
     def create
-      total_prizes_percentages = Prize.pluck(:percentage).sum
-      if total_prizes_percentages + params[:prize][:percentage].to_i > 100
+      if Prize.pluck(:percentage).sum + params[:prize][:percentage].to_i > 100
         render json: 'Error! The total percentage is greater than 100%!'
       else
-        Prize.create(prize_params)
-        redirect_to '/admin/prizes'
-      end
+        Prize.create(params.require(:prize).permit(:name, :percentage, :image, :prize_type, :description, :identifier))
+        redirect_to '/admin/prizes' end
     end
 
     def update
-      prize_info = Prize.find_by(id: params[:id])
-      total_prizes_percentages = Prize.pluck(:percentage).sum - prize_info.percentage
-
-      if total_prizes_percentages + params[:prize][:percentage].to_i > 100
+      if (Prize.pluck(:percentage).sum - Prize.find_by(id: params[:id]).percentage) +
+         params[:prize][:percentage].to_i > 100
```

Comment:
To be honest I think what you had before was better... Not every piece of code needs to be optimized to be the smallest possible but to be the easiest to understand, and I think that before was easier in that sense.

Reply:
Same reason, I optimize the code because I was having this error: `Block has too many lines. [35/25] (convention:Metrics/BlockLength)`. Can I fix it with this `# rubocop:disable Metrics/BlockLength`?